composer.json

```diff
@@ -15,7 +15,7 @@
     "canyon-gbs/authorization": "*",
     "canyon-gbs/assist-data-model": "*",
     "canyon-gbs/case": "*",
-    "filament/filament": "v3.0.0-beta11",
+    "filament/filament": "^3.0.0",
```

Comment:
🎉

Reply:
I believe I may have done this already in a recent commit I pushed up, I'll check though.